test/unit/test-amp-story-player.js

```diff
@@ -39,6 +43,22 @@ describes.realWin('AmpStoryPlayer', {amp: false}, env => {
 
   beforeEach(() => {
     win = env.win;
+
+    fakeMessaging = {
+      setDefaultHandler: () => {},
+      registerHandler: (event, handler) => {
+        fireHandler = handler;
+      },
+    };
+    fakeMessagingPromise = new Promise(resolve => {
```

Comment:
`*.resolves(foo)` is a shorthand for `*.returns(Promise.resolve(foo))`
You should be able to directly call `.resolves(fakeMessaging)` without this extra variable :)